src/main/java/net/neoforged/neoforge/common/IOUtilities.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common;
+
+import java.io.BufferedOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.function.BiPredicate;
+import java.util.function.Consumer;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.NbtIo;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Declares a class containing helpers for performing file I/O in a resillient
+ * manner.
```

Comment:
Remove the "Declares a class containing", and substitute with "Contains".
```suggestion
 * Contains helpers for performing file I/O in a resillient manner.
```

Reply:
:thinking: I don't even know why I wrote it this way in the first place, you're right.

Reply:
Isn't the standard spelling `resilient`?

Reply:
Yes; this was a typo.